Neos.Fusion/Tests/Functional/FusionObjects/DataStructureTest.php

```diff
@@ -139,10 +139,40 @@ public function untypedChildKeysWorkWithFusionPositionSorting(): void
     /**
      * @test
      */
-    public function unsetUntypedChildKeysWillRenderAsDataStructure(): void
+    public function unsetChildKeyWillNotRender(): void
     {
         $view = $this->buildView();
-        $view->setFusionPath('dataStructure/unsetUntypedChildKeyWillRenderAsDataStructure');
-        self::assertEquals(['buz' => 456, 'keyWithUnsetType' => ['bat' => 123]], $view->render());
+        $view->setFusionPath('dataStructure/unsetChildKeyWillNotRender');
+        self::assertEquals(['foo' => 'bar'], $view->render());
+    }
+
+    /**
+     * @test
+     */
+    public function unsetUntypedChildKeyWillNotRender(): void
+    {
+        $view = $this->buildView();
+        $view->setFusionPath('dataStructure/unsetUntypedChildKeyWillNotRender');
+        self::assertEquals(['buz' => 456], $view->render());
+    }
```

Comment:
**NOTE**: This test is modified.
The corresponding Fusion code is:

```fusion
dataStructure.unsetUntypedChildKeyWillNotRender = Neos.Fusion:DataStructure {
  keyWithUnsetType = Neos.Fusion:Value {
    value = 123
  }
  keyWithUnsetType >
  keyWithUnsetType {
    bat = 123
  }
  buz = 456
}
```

Previously the test `unsetUntypedChildKeysWillRenderAsDataStructure` expected the result to be:
```json
{"buz": 456, "keyWithUnsetType": {"bat": 123}}
```
Now it expects:
```json
{"buz": 456}
```

which is in sync with Neos 7.x behavior